libs/modal/.storybook/preview.tsx

```diff
@@ -15,7 +16,7 @@ const preview: Preview = {
   decorators: [
     (Story, context) => {
       if (context.id === "modal--primary") {
-        return renderStoryWithContainer(osmosisTheme);
+        return renderStoryWithContainer(ztxTheme);
```

Comment:
Why are you changing the existing osmosis story? Add a new story for ztx, so that our Osmosis Demo still continues to work.

Reply:
I only wanted to see it alone in the primary for my development, I can change it back.

Reply:
You should, and you should create a new story for ztx. Please be more careful about such changes in the future, you shouldn't rely on me catching that in a code review.